.mailmap

```diff
@@ -20,4 +20,5 @@ Tobias Feldballe <tobias@osandweb.dk> <tobi.fp@gmail.com>
 Tobias Feldballe <tobias@osandweb.dk> <tf@jumpstory.com>
 Jonas Borchelt <jonas.borchelt@connected.link>
 Derek Frogget <fro22003@byui.edu> <114030121+derekfrogget@users.noreply.github.com>
-Nikhil Ghosh <nghosh606@gmail.com>
\ No newline at end of file
+Nikhil Ghosh <nghosh606@gmail.com>
+David Wever <dmwever@crimson.ua.edu> <56411717+dmwever@users.noreply.github.com>
```

Comment:
you don't need this entry if you update and fix your mail/author name in all the commits of this pull request. since they are not merged yet, this can easily be done - the file is for commits that have been merged already :)

Reply:
I think both mails are in the commits. The GitHub one is used when you apply suggestions to PRs in the web interface.